script/DeployBN254EncryptionOracle.s.sol

```diff
@@ -11,11 +11,14 @@ contract DeployBN254EncryptionOracle is BaseScript {
     function run() external {
         uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
         address relayer = vm.envAddress("NODE_1_ADDRESS");
+        uint96 oracleFee = uint96(vm.envUint("ORACLE_FEE"));
```

Comment:
Can we also be flexible in terms of the fee for the submission and the fee for the request ? The 2x is fixed and does not give us great flexibility. 
What do you think about changing this variable to two `SUBMISSION_FEE`and `REENCRYPTION_FEE` and we set manually those two ? it doesn't seem that much manual work for us when we deploy and gives us much more flexbility down the line.